ibm/service/kubernetes/resource_ibm_container_alb.go

```diff
@@ -102,28 +124,17 @@ func resourceIBMContainerALBCreate(d *schema.ResourceData, meta interface{}) err
 		return err
 	}
 	var userIP string
-	var enable, disableDeployment bool
+	var enable bool
 	albID := d.Get("alb_id").(string)
-	if v, ok := d.GetOkExists("enable"); ok {
-		enable = v.(bool)
-	} else if v, ok := d.GetOkExists("disable_deployment"); ok {
-		disableDeployment = v.(bool)
-	} else {
-		return fmt.Errorf("[ERROR] Provide either `enable` or `disable_deployment`")
+	v, ok := d.GetOkExists("enable")
+	if !ok {
+		return fmt.Errorf("[ERROR] Missing `enable` argument")
 	}
+	enable = v.(bool)
 
-	numOfInstances := "2"
 	if v, ok := d.GetOk("user_ip"); ok {
```

Comment:
This is also only relevant if `enable == true`, should be moved to the if condition.